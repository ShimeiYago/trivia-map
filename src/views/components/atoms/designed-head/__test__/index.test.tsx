import { shallow, ShallowWrapper } from 'enzyme';
import { DesignedHead } from '..';

let wrapper: ShallowWrapper;

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<DesignedHead children="xxx" />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('basic', () => {
    wrapper.setProps({
      component: 'h5',
    });
    expect(wrapper).toMatchSnapshot();
  });
});
