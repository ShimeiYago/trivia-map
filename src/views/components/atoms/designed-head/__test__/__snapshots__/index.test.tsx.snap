// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<ForwardRef(Typography)
  component="h2"
  sx={
    Object {
      "borderBottom": "solid 2px #7db4e6",
      "borderLeft": "solid 5px #7db4e6",
      "mb": 2,
      "mt": 1,
      "px": 1,
      "py": 0.5,
    }
  }
  variant="h5"
>
  xxx
</ForwardRef(Typography)>
`;
